phil/cnx-routes.md

```diff
@@ -0,0 +1,302 @@
+# CNX APIs
+
+https://cnx.org APIs are split into 4 areas and documented in greater detail in each repository.
+
+You can retrieve published content using the [^cnx-archive] APIs (including search), and can change content using the [^cnx-publishing-api] APIs.
+
+Browser URLs are managed and documented in [^webview].
+
+## Identifiers
+
+There are 5 types of identifiers used within https://cnx.org
+
+- [sha hash](#sha-hash)
+- [UUID](#uuid)
+- [Short UUID](#short-uuid)
+- [UUID and version](#uuid-and-version)
+- [Short UUID and version](#short-uuid-and-version)
+
+### Sha hash
+
+These are used for identifying resources (ie images) and the hash is a hash of the bits in the resource file.
```

Comment:
s/ie/e.g./  - _i.e._ implies an alternative (i.e. == "that is") while _e.g._ == _exampli gratia_  or "for example"
the value is the sha1 hash (as provided by sha1sum or sha1) of the resource file bits.